.circleci/config.yml

```diff
@@ -142,13 +185,13 @@ jobs:
             - version-to-deploy.txt
 
   build_production:
-    <<: *defaults
+    executor: build_executor
     steps:
       - checkout
       - setup_remote_docker:
           docker_layer_caching: true
       - restore_cache:
-          key: dependency-cache-{{ checksum "package-lock.json" }}
+          key: dependency-cache-1-{{ checksum "package-lock.json" }}
```

Comment:
we can update this back too something sensible once we know package lock has changed